draft-ietf-ppm-dap-taskprov.md

```diff
@@ -686,8 +716,8 @@ that define:
 1. A new Taskbind extension
 
 Documents defining either a new DAP batch mode or VDAF SHOULD include a section
-titled "Taskbind Considerations" that specifies how to extend the `BatchConfig`
-structure and the `VdafConfig` structure respectively.
+titled "Taskbind Considerations" that specifies the payload of
+`TaskConfig.batch_config` or `TaskConfig.vdaf_config` respectively.
```

Comment:
This change is the main pay off of this PR. "Extend the `VdafConfig` structure" is not very well defined.

Reply:
So do we have any language on what if `batch_mode` or `vdaf_type` is unrecognized by an aggregator? (which is the initial motivation to make `vdaf_config` an opaque)

Reply:
Yes, if not recognized then the party MUST opt out.